indico/modules/events/editing/client/js/editing/page_layout/MenuBar.jsx

```diff
@@ -6,58 +6,83 @@
 // LICENSE file for more details.
 import managementURL from 'indico-url:event_management.settings';
 import displayURL from 'indico-url:events.display';
+import editableTypeListURL from 'indico-url:event_editing.editable_type_list';
 
 import React from 'react';
+import {Link} from 'react-router-dom';
 import PropTypes from 'prop-types';
-import {Header, Icon} from 'semantic-ui-react';
+import {Header, Icon, Menu} from 'semantic-ui-react';
 import {Translate} from 'indico/react/i18n';
+import Palette from 'indico/utils/palette';
+import {EditableEditingTitles} from '../../models';
 
 import './MenuBar.module.scss';
 
-export default function MenuBar({eventId, menuItems}) {
+function EditableListMenu({eventId, type}) {
+  if (location.pathname === editableTypeListURL({confId: eventId, type})) {
+    return (
+      <Menu vertical>
+        <Menu.Item active>
+          <Translate>Editable list</Translate>
+        </Menu.Item>
+      </Menu>
+    );
+  }
+  return (
+    <Menu vertical>
+      <Menu.Item>
+        <Link to={editableTypeListURL({confId: eventId, type})}>
+          <Translate>Editable list</Translate>
+        </Link>
+      </Menu.Item>
+    </Menu>
+  );
+}
+
+EditableListMenu.propTypes = {
+  eventId: PropTypes.number.isRequired,
+  type: PropTypes.string.isRequired,
```

Comment:
`editableType`?

and i'd use `PropTypes.oneOf(...).isRequired` since we have a well-defined list of editable types